test/controllers/RequestMetadataReviewControllerSpec.scala

```diff
@@ -55,6 +55,7 @@ class RequestMetadataReviewControllerSpec extends FrontEndTestHelper {
       playStatus(content) mustBe OK
       contentType(content) mustBe Some("text/html")
       requestMetadataReviewPageAsString must include("<title>Request a metadata review - Transfer Digital Records - GOV.UK</title>")
+      requestMetadataReviewPageAsString must include(s"""<a href="/consignment/$consignmentId/additional-metadata/download-metadata" class="govuk-back-link">Back</a>""")
       requestMetadataReviewPageAsString must include(
```

Comment:
Can you add a check for back link on the `metadata-review/submit-request` route goes to the `requestMetadataReview` page.

Reply:
Actually, `requestMetadataReview` page has not been developed fully yet as it is out of the scope of my ticket. I just created it as a template.